### PR TITLE
[chrome-ext] expand the list of puzzles to fill the full screen in th…

### PR DESCRIPTION
**Pull request by jeevnayak**
…e dashboard

- move the chart to a collapsible overlay

### CODE REVIEW

File: packages/chrome-extension/src/dashboard/DashboardRoot.tsx [hunk 1]
```diff
@@ -8,21 +8,38 @@ import SolveProgressChart from "./charts/SolveProgressChart";
...
+                        <div
+                            className="DashboardRoot-closeChart"
+                            onClick={() => this.setState({showChart: false})}
+                        >
+                            <svg className="DashboardRoot-closeIcon" viewBox="0 0 32 32">
+                                <polygon points="24.485,27.314 27.314,24.485 18.828,16 27.314,7.515 24.485,4.686 16,13.172 7.515,4.686 4.686,7.515 13.172,16 4.686,24.485 7.515,27.314 16,18.828 " />
+                            </svg>
+                        </div>
+                    </div>
+                    <div className="DashboardRoot-chartContainer">
```
Comment (vsiao) [inline packages/chrome-extension/src/dashboard/DashboardRoot.tsx]: do you need the container div? you can probably just transfer its styles to `-chart`?

File: packages/chrome-extension/src/dashboard/DashboardRoot.scss [hunk 2]
```diff
@@ -29,12 +29,46 @@
...
+}
+
+.DashboardRoot-chartHeader {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.DashboardRoot-closeChart {
+  padding-top: 7px;
+  padding-right: 7px;
```
Comment (vsiao) [inline packages/chrome-extension/src/dashboard/DashboardRoot.scss]: let's either do padding all the way around, or change these to margin

File: packages/chrome-extension/src/dashboard/DashboardRounds.scss [hunk 3]
```diff
@@ -2,6 +2,7 @@
   background: #003541;
   display: flex;
   flex-basis: 50vh;
+  flex-grow: 1;
   min-height: 300px;
```
Comment (vsiao) [inline packages/chrome-extension/src/dashboard/DashboardRounds.scss]: can drop `flex-basis` and `min-height` now that we're just filling the whole screen

File: packages/chrome-extension/src/dashboard/DashboardRoot.scss (hunk 2 above)
Comment (jeevnayak) [reply packages/chrome-extension/src/dashboard/DashboardRoot.scss]: changed to margin

File: packages/chrome-extension/src/dashboard/DashboardRoot.tsx (hunk 1 above)
Comment (jeevnayak) [reply packages/chrome-extension/src/dashboard/DashboardRoot.tsx]: good call, it was somehow broken when i was messing with it before but now it works

**Comment by jeevnayak** [comment]
image

**Pull request merged**